views/components/content-tabs/macro.njk

```diff
@@ -3,10 +3,10 @@
 {% macro contentTabs(tabs) %}
     {# data-breakpoint must match breakpoint used for styles #}
     <div class="js-tab-container" data-breakpoint="(min-width: 620px)">
-        <ul role="tablist" class="js-tabset content-tabset">
```

Comment:
This was being flagged as an invalid combination of roles when I removed the rule exclusions in the aXe config. Doing a bit of digging it sounds like you have to be careful with the `tablist` role and it's better to leave as a plain list.